framework/src/engine/rpc/rpc_server.ts

```diff
@@ -231,7 +233,7 @@ export class RPCServer {
 
 		try {
 			const request = Request.fromJSONRPCRequest(requestObj);
-			if (this._isDisabledMethod(request.namespace, request.name)) {
+			if (this._isAllowedMethod(request.namespace, request.name)) {
```

Comment:
```suggestion
			if (!this._isAllowedMethod(request.namespace, request.name)) {
```
i think this should be negated?